<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1460</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simple语言解释器【作者：重庆理工大学-王淞-11903990330】</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon1/程序图标.svg</normaloff>icon1/程序图标.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="edit_tabWidget">
         <property name="styleSheet">
          <string notr="true">font-size:12pt</string>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="encoding_label">
         <property name="styleSheet">
          <string notr="true">font-size:10pt</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1460</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopenworkspace"/>
    <addaction name="actionsave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhelp"/>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>程序分析</string>
    </property>
    <addaction name="actionanalysis"/>
    <addaction name="actiongrammar_run"/>
    <addaction name="actioninter"/>
    <addaction name="actionrun"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionopenworkspace"/>
   <addaction name="actionsave"/>
   <addaction name="actionanalysis"/>
   <addaction name="actiongrammar_run"/>
   <addaction name="actioninter"/>
   <addaction name="actionrun"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>229</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:12pt</string>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>文件目录</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>386</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:12pt</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>程序分析</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true">font-size:12pt</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>词法分析</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>0</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>50</number>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>行号  </string>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>单词</string>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>状态码</string>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>类别</string>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTableWidget" name="tableWidget_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>行号</string>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>错误</string>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <italic>false</italic>
              <underline>false</underline>
             </font>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>语法分析</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTextEdit" name="textEdit_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>生成语法树</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>中间代码</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tableWidget_3">
           <column>
            <property name="text">
             <string>No</string>
            </property>
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
           </column>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>2</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>3</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>4</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionhelp">
   <property name="icon">
    <iconset>
     <normaloff>icon1/帮助.svg</normaloff>icon1/帮助.svg</iconset>
   </property>
   <property name="text">
    <string>帮助文档（H）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="icon">
    <iconset>
     <normaloff>icon1/关于我们.svg</normaloff>icon1/关于我们.svg</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionopenworkspace">
   <property name="icon">
    <iconset>
     <normaloff>icon1/新增文件夹.svg</normaloff>icon1/新增文件夹.svg</iconset>
   </property>
   <property name="text">
    <string>打开工作区</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset>
     <normaloff>icon1/保存.svg</normaloff>icon1/保存.svg</iconset>
   </property>
   <property name="text">
    <string>保存（S）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionanalysis">
   <property name="icon">
    <iconset>
     <normaloff>icon1/词法分析.svg</normaloff>icon1/词法分析.svg</iconset>
   </property>
   <property name="text">
    <string>词法分析</string>
   </property>
  </action>
  <action name="actiongrammar_run">
   <property name="icon">
    <iconset>
     <normaloff>icon1/语法分析.svg</normaloff>icon1/语法分析.svg</iconset>
   </property>
   <property name="text">
    <string>语法分析</string>
   </property>
  </action>
  <action name="actioninter">
   <property name="icon">
    <iconset>
     <normaloff>icon1/中间代码.svg</normaloff>icon1/中间代码.svg</iconset>
   </property>
   <property name="text">
    <string>中间代码</string>
   </property>
  </action>
  <action name="actionrun">
   <property name="icon">
    <iconset>
     <normaloff>icon1/运行，调试.svg</normaloff>icon1/运行，调试.svg</iconset>
   </property>
   <property name="text">
    <string>程序运行</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
